src/utils/ResultListUtils.ts

```diff
@@ -0,0 +1,24 @@
+import { Component } from '../ui/Base/Component';
+import { ResultList } from '../ui/ResultList/ResultList';
+import { $$ } from './Dom';
+import { find } from 'underscore';
+
+export class ResultListUtils {
+  public static isInfiniteScrollActive(root: HTMLElement) {
+    const resultList = ResultListUtils.getActiveResultList(root);
+    return resultList ? !!resultList.options.enableInfiniteScroll : false;
+  }
+
+  private static getActiveResultList(root: HTMLElement) {
+    const resultLists = ResultListUtils.getResultLists(root);
+    return find(resultLists, resultList => !resultList.disabled);
+  }
+
+  private static getResultLists(root: HTMLElement) {
```

Comment:
I'd add those to `src/utils/ComponentsTypes` instead.

Reply:
I moved this method over but decided to revert it. `ComponentsTypes` atm only holds methods for facets, while this is for ResultLists. I think it is more cohesive to keep the method with other ResultList operations. I would also suggest renaming `ComponentsTypes` to `FacetUtils`.

What do you think?